exercises/C-variables/exercise.js

```diff
@@ -1,8 +1,7 @@
 // Program to print a `greeting` variable in the console 3 times
-// Hopefully it is ok to be a bit creative with my solution and reduce code repetition :)
+// Tried to reduce code repetition by using a for loop
 
-let greeting =
-  ". My name is Andrew and Ive been learning how to code with <Code>YourFuture since 10/04/21";
+let greeting = ". Hello world";
```

Comment:
Why there is dot before hello world text ?

Reply:
Thanks for the feedback Ibrahim,

By using a for loop to create the three console log messages I was able to then use its count variable "i"  to add a preceding number to each message and display,

1. Hello world
2. Hello world
3. Hello world

So, the second character in the above messages is the dot before the hello world text :)